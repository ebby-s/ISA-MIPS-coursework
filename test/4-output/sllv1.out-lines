TB : INFO : register_v0=00000020
TB : finished; active=0
