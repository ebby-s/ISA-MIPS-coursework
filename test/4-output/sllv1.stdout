Loading test/2-binary/sllv1.hex.txt into RAM
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
TB : INFO : register_v0=00000020
TB : finished; active=0
